test/tests_for_utils/catalog_cache_test.py

```diff
@@ -0,0 +1,71 @@
+# This test only tests code that can be exercised without a network connection to services.
+# That code is tested in integration tests.
+import pytest
+
+from unittest.mock import create_autospec, MagicMock
+
+from installed_clients.CatalogClient import Catalog
+from execution_engine2.utils.catalog_cache import CatalogCache
+
+
+@pytest.fixture
+def catalog():
+    catalog = create_autospec(Catalog, spec_set=True, instance=True)
+    catalog.list_client_group_configs = MagicMock(
+        return_value={
+            "module_name": "module_name",
+            "function_name": "function_name",
+            "client_groups": "client_groups",
+        }
+    )
+    catalog.get_module_version = MagicMock(return_value={"git_commit_hash": 123})
+
+    return catalog
+
+
+def test_cc_job_reqs(catalog):
+    """Test to see the job requirements cache is being used."""
```

Comment:
For this and the next test, should test that the returned value is as expected

Reply:
Isn't it contrived, since the returned value is mocked in the test? I don't see the benefit of doing it. I have updated it, can you please re-review it?

Reply:
You still want to test that the cache returns the expected value, right? If someone accidentally ctnl-Ds the return line and the tests still pass that's probably not what you want

Reply:
The changes look good, but I realized there's nothing testing that the service version is being supplied to the catalog client as expected

Reply:
Oh I see. I have added them now

Reply:
:+1: